modules/core/04-channel/keeper/timeout.go

```diff
@@ -156,6 +156,7 @@ func (k Keeper) TimeoutExecuted(
 	if channel.Ordering == types.ORDERED {
 		channel.State = types.CLOSED
 		k.SetChannel(ctx, packet.GetSourcePort(), packet.GetSourceChannel(), channel)
+		EmitChannelClosedEvent(ctx, packet, channel)
```

Comment:
@colin-axner recommended in his [comment](https://github.com/cosmos/ibc-go/issues/202#issuecomment-1123647973) to do the event emission after `EmitTimeoutPacketEvent(ctx, packet, channel)` below. Would you mind moving it there? I guess you need to wrap it in a check to make sure that the channel is ordered and is closed...

Reply:
Ah I missed that part of the comment, thanks @crodriguezvega, I've made the appropriate changes

Reply:
Thanks, @chatton!

Reply:
Just as a note, I actually disagree with my past comment. Emitting after closing the channel works just fine imo